docs/agent-vs-aea.md

```diff
@@ -6,7 +6,7 @@ In this guide we show some of the differences in terms of code.
 
 The <a href="../build-aea-programmatically">Build an AEA programmatically</a> guide shows how to programmatically build an AEA. We can build an agent of the <a href="../api/agent#agent-objects">`Agent`</a> class programmatically as well.
 
-First, import the python and application specific libraries. (Get the packages directory from the AEA repository `svn export https://github.com/valory-xyz/open-aea.git/trunk/packages`.)
+First, import the python and application specific libraries. (Get the packages directory from the AEA repository `aea fetch PUBLIC_ID_OR_HASH`.)
```

Comment:
```suggestion
First, import the python and application specific libraries. (Get the packages directory from the AEA repository `svn export https://github.com/valory-xyz/open-aea.git/trunk/packages`.)
```
It makes no sense here now. If you read the sentence it has no place here as such. The whole guide doesn't make any sense anymore. It needs to be properly fixed.